Library/ViewModels/ProjectPamphletMainCellViewModelTests.swift

```diff
@@ -507,8 +507,8 @@ final class ProjectPamphletMainCellViewModelTests: TestCase {
 
     self.notifyDelegateToGoToCreator.assertValues([project])
 
-    XCTAssertEqual(["CTA Clicked"], self.dataLakeTrackingClient.events)
-    XCTAssertEqual(["CTA Clicked"], self.segmentTrackingClient.events)
+    XCTAssertEqual(["Page Viewed", "CTA Clicked"], self.dataLakeTrackingClient.events)
```

Comment:
Could you explain why this changed? This method is for the goToCreator event which you did not change in this PR, right?

Reply:
Because trigger location for `projectViewed` has changed from when the button is tapped, when the cell is loaded due to `awakeFromNib()` there will already be a `Page Viewed` event before the CTA Clicked event.